server/data/data-jmap/src/main/java/org/apache/james/jmap/api/vacation/Vacation.java

```diff
@@ -105,6 +105,21 @@ public String getTextBody() {
         return textBody;
     }
 
+    public boolean isActiveAtDate(ZonedDateTime zonedDateTime) {
```

Comment:
the argument name adds nothing, you already have the type for that. You probably can name it `instant` without losing anything